cdk-infra/.gitignore

```diff
@@ -8,4 +8,6 @@ node_modules
 cdk.out
 
 # Build artifacts
-dist
\ No newline at end of file
+dist
+
+cdk.context.json
```

Comment:
Not relevant to the current PR, but I'm adding it now since I want to make sure I don't accidentally commit this file.